benches/buffering.rs

```diff
@@ -1,25 +1,19 @@
 use criterion::{criterion_group, criterion_main, Benchmark, Criterion, Throughput};
 
 use futures::compat::Future01CompatExt;
-use rand::{rngs::SmallRng, thread_rng, Rng, SeedableRng};
-use rand_distr::Alphanumeric;
 use tempfile::tempdir;
 use vector::test_util::{
-    next_addr, runtime, send_lines, start_topology, wait_for_tcp, CountReceiver,
+    next_addr, random_lines, runtime, send_lines, start_topology, wait_for_tcp, CountReceiver,
 };
 use vector::{buffers::BufferConfig, config, sinks, sources};
 
 fn benchmark_buffers(c: &mut Criterion) {
-    let num_lines: usize = 100_000;
+    let num_lines: usize = 10_000;
```

Comment:
I feel like I'm repeating myself, but this is a very coarse grained measurement and criterion is already doing several thousand iterations for us.